integration/log_integration_test.go

```diff
@@ -45,9 +46,23 @@ type testParameters struct {
 	sequencingPollWait  time.Duration
 }
 
+type consistencyProofParams struct {
+	firstTreeSize  int64
```

Comment:
(style) Go tends towards terse names, so the type and field names here seem a bit long for something that's only used in this file.